docs/conf.py

```diff
@@ -28,9 +28,11 @@
 pygments_style = 'sphinx'
 
 intersphinx_mapping = {
-    'http://docs.python.org/2.7': None,
-    'http://docs.djangoproject.com/en/1.9':
+    'python': ('http://docs.python.org/3', None),
+    'django': (
+        'http://docs.djangoproject.com/en/1.9',
         'http://docs.djangoproject.com/en/1.9/_objects/',
```

Comment:
Does this have to be an outdated version of Django?

Reply:
Not particularly.  Fixed.